templates/xml/compound.xsd

```diff
@@ -764,7 +764,7 @@
       <xsd:enumeration value="PHP" />
       <xsd:enumeration value="Objective-C" />
       <xsd:enumeration value="C++" />
-      <xsd:enumeration value="Javascript" />
+      <xsd:enumeration value="JavaScript" />
```

Comment:
I hope nothing outside this project cares.

Reply:
Not sure about this. @morsa would this hit you?